src/fs-helpers.ts

```diff
@@ -39,17 +51,36 @@ export function jsonFile<T>(path: string, obj: T) {
   };
   return file;
 }
+function cloneFile(f: File): File {
+  if(f.type === 'json') {
+    return jsonFile(f.path, f.obj);
```

Comment:
Should we JSON.stringify/parse here? Otherwise could clone, mutate, and the original would be changed.

Reply:
Doh, and that is exactly the use-case I was hoping to support here: cloning and then mutating the object.

Fixed